docs/book/src/intro/installation.md

```diff
@@ -65,23 +65,23 @@ anvil-zksync
 
 These URLs always fetch the **latest** published build:
 
-- [`anvil-zksync-aarch64-apple-darwin.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.5.1-aarch64-apple-darwin.tar.gz)
-- [`anvil-zksync-aarch64-unknown-linux-gnu.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.5.1-aarch64-unknown-linux-gnu.tar.gz)
-- [`anvil-zksync-x86_64-apple-darwin.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.5.1-x86_64-apple-darwin.tar.gz)
-- [`anvil-zksync-x86_64-unknown-linux-gnu.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.5.1-x86_64-unknown-linux-gnu.tar.gz)
+- [`anvil-zksync-aarch64-apple-darwin.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.6.0-aarch64-apple-darwin.tar.gz)
+- [`anvil-zksync-aarch64-unknown-linux-gnu.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.6.0-aarch64-unknown-linux-gnu.tar.gz)
+- [`anvil-zksync-x86_64-apple-darwin.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.6.0-x86_64-apple-darwin.tar.gz)
+- [`anvil-zksync-x86_64-unknown-linux-gnu.tar.gz`](https://github.com/matter-labs/anvil-zksync/releases/latest/download/anvil-zksync-v0.6.0-x86_64-unknown-linux-gnu.tar.gz)
 
 ## 4. Docker
 
 `anvil-zksync` is available as Docker image that supports Linux x64 and arm64.
 
 ```bash
-docker pull ghcr.io/matter-labs/anvil-zksync:v0.5.1
+docker pull ghcr.io/matter-labs/anvil-zksync:v0.6.0
```

Comment:
Same here I am guessing - `latest` should default to the latest stable release. Right now we likely overwrite `latest` on every commit to main

Reply:
Created a task to track: https://github.com/matter-labs/anvil-zksync/issues/693